code/index.html

```diff
@@ -2,21 +2,152 @@
+            <h1>Article 2</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>      
+        <div class="card">
+          <div class="article-3">
+            <a href="#">
+            <img src="./img/img-3.jpg" alt="succulents image">
+            <h1>Article 3</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="article-4">
+            <a href="#">
+            <img src="./img/img-4.jpg" alt="succulents image">
+            <h1>Article 4</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div> 
+        <div class="card">
+          <div class="article-5">
+            <a href="#">
+            <img src="./img/img-5.jpg" alt="succulents image">
+            <h1>Article 5</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="article-6">
+            <a href="#">
+            <img src="./img/img-6.jpg" alt="succulents image">
+            <h1>Article 6</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>                 
+        <div class="card">
+          <div class="article-7">
+            <a href="#">
+            <img src="./img/img-7.jpg" alt="succulents image">
+            <h1>Article 7</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>          
+        <div class="card">
+          <div class="article-8">
+            <a href="#">
+            <img src="./img/img-8.jpg" alt="succulents image">
+            <h1>Article 8</h1>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem debitis nesciunt eum accusamus corrupti voluptates officiis.</p>
+            </a>
+            <a href= "#" class="links">Read more</a>
+          </div>
+        </div>
+      </div>  
+    </section>
     </section>
+    </main>
```

Comment:
Indentation again, should look something like this:
```
<main>
  <section>
    <section>
    </section>
  </section>
</main>
```
so one new level of indentation for each level of nesting. Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) again if you're unsure :)